### PR TITLE
More code reformatting:

### PR DESCRIPTION
**Pull request by erikma**
For src\tasks\e*-q*.cs
- Code style updates (mostly automated)
- Reduced .NET 1.x data structure use to reduce up- and down-casting
- Reduced allocations in a couple of spots

### CODE REVIEW

**Comment by erikma** [comment]
For #3320

**Comment by AndyGerlicher** [comment]
@dotnet-bot test Ubuntu16.04 Build for CoreCLR please

@rainersigwald uhh any idea how `Ubuntu14.04 Build for CoreCLR` could still be queuing to run?

File: src/Tasks/FindInList.cs [hunk 1]
```diff
@@ -18,17 +18,12 @@ public class FindInList : TaskExtension
         // The list to search through
         private ITaskItem[] _list;
         // Whether to match just the file part, or the full item spec
-        private bool _matchFileNameOnly = false;
```
Comment (cdmihai) [inline src/Tasks/FindInList.cs]: Bunch of dangling comments wherever this refactoring ran. Sad the duplicated comments were there in the first place.

Comment (erikma) [reply src/Tasks/FindInList.cs]: Fixed in following PR coming today/tomorrow, if that's OK

**Pull request merged**